python/ray/__init__.py

```diff
@@ -109,7 +109,6 @@
     "get_runtime_context",
     "actor",
     "actors",
-    "workers",
```

Comment:
Why this change?

Reply:
Because it's broken, as I asked about the other day. There's no import in this file that satisfies `ray.workers()` -- so it's always been broken when called. So rather than expose something that never worked, I cleaned it up as I went by.